risc0/build/src/docker.rs

```diff
@@ -154,14 +154,19 @@ fn create_dockerfile(
     .join(" ");
 
     let build = DockerFile::new()
-        .from_alias("build", "risczero/risc0-guest-builder:r0.1.79.0-2")
+        .from_alias("build", "risczero/risc0-guest-builder:r0.1.79.0-3")
         .workdir("/src")
         .copy(".", ".")
         .env(manifest_env)
         .env(rustflags_env)
         .env(&[("CARGO_TARGET_DIR", "target")])
         // Fetching separately allows docker to cache the downloads, assuming the Cargo.lock
         // doesn't change.
+        .env(&[(
+            "CC",
+            "/root/.local/share/cargo-risczero/cpp/bin/riscv32-unknown-elf-gcc",
+        )])
```

Comment:
I would imagine we'd only want to configure the guest-side, this will also include the host-side.

Reply:
Good point - we don't have any spaces in the gcc path so I pushed a change just now to use the `CC_riscv32im_risc0_zkvm_elf` flag to keep it confined to the guest